snapshot/lib/assets/SnapfileTemplate

```diff
@@ -2,11 +2,25 @@
 
 # A list of devices you want to take the screenshots from
 # devices([
+#   "iPhone 5",
+#   "iPhone 5s",
 #   "iPhone 6",
 #   "iPhone 6 Plus",
-#   "iPhone 5",
-#   "iPad Pro (12.9 inch)",
-#   "iPad Pro (9.7 inch)"
+#   "iPhone 6s",
+#   "iPhone 6s Plus",
+#   "iPhone 7",
+#   "iPhone 7 Plus",
+#   "iPhone SE",
+#   "iPad Retina",
+#   "iPad Air",
```

Comment:
I don't think we want to include that many devices by default. Just adding the `Apple TV 1080p` option should be fine.